test/unit/job_template_test.rb

```diff
@@ -109,6 +109,67 @@
     end
   end
 
+  context 'importing an existing template' do
+    let(:existing) do
+      template = <<-END_TEMPLATE
+      <%#
+      kind: job_template
+      name: Ping a Thing
+      job_category: Commands
+      provider_type: SSH
+      template_inputs:
+      - name: hostname
+        input_type: user
+        options: "www.google.com"
+        required: true
+      %>
+
+      ping -c 5 <%= input("hostname") %>
+      END_TEMPLATE
+
+      JobTemplate.import(template, :default => true)
+    end
+
+    let(:updated) do
+      <<-END_TEMPLATE
+      <%#
+      kind: job_template
+      name: Ping a Thing
+      job_category: Commands
+      provider_type: SSH
+      template_inputs:
+      - name: hostname
+        input_type: user
+        options: "www.google.com\nwww.redhat.com"
+        required: true
+      - name: count
+        input_type: user
+        required: true
+      %>
+
+      ping -c <%= input("count") %> <%= input("hostname") %>
+      END_TEMPLATE
+    end
+
+    it 'will not overwrite by default' do
+      refute existing.import!(updated)
+    end
+
+    let(:synced_template) do
+      existing.import!(updated, :update => true)
```

Comment:
@iNecas  Updated.  Local tests won't run on my Katello for some reason, so if you're really anxious to get this merged (seems like you're watching closely?), then if you want to run job_template_test again locally, by all means....